### PR TITLE
Add in Developer Blog announcing Materialized Views

### PR DESCRIPTION
**Pull request by amychen1776**
## What are you changing in this pull request and why?
I'm adding a new devblog post for materialized views 

Adding new pages (delete if not applicable):
- [ ] Add page to `website/sidebars.js`
- [x ] Provide a unique filename for the new page

### CODE REVIEW

File: website/blog/2023-08-01-announcing-materialized-views.md
Comment (jasnonaz) [inline website/blog/2023-08-01-announcing-materialized-views.md]: In general we try and avoid framing dev blog posts as _announcements_. Can we frame this around why people would find these useful?

File: website/blog/2023-08-01-announcing-materialized-views.md
Comment (jasnonaz) [inline website/blog/2023-08-01-announcing-materialized-views.md]: We'll want to update this to the actual go-live date

File: website/blog/2023-08-01-announcing-materialized-views.md
Comment (jasnonaz) [inline website/blog/2023-08-01-announcing-materialized-views.md:21]: I'm sure the answer is yes but just to doublecheck - Jetblue is ok with us sharing this?

File: website/blog/2023-08-01-announcing-materialized-views.md
Comment (jasnonaz) [inline website/blog/2023-08-01-announcing-materialized-views.md]: Should we link these docs?

File: website/blog/2023-08-01-announcing-materialized-views.md
Comment (jasnonaz) [inline website/blog/2023-08-01-announcing-materialized-views.md:213]: Do we want to send them somewhere for feedback?

**Review by jasnonaz — CHANGES_REQUESTED** [review]
This is great!! I have a couple logistical notes and small questions, but other than those this feels good to go from my end.

File: website/blog/2023-08-01-announcing-materialized-views.md
Comment (amychen1776) [reply website/blog/2023-08-01-announcing-materialized-views.md]: Makes sense - I can do that once it's approved?

File: website/blog/2023-08-01-announcing-materialized-views.md
Comment (amychen1776) [reply website/blog/2023-08-01-announcing-materialized-views.md]: How about- 

`We have Materialized Views: What do we do now?` 

or

`How to not f*ck up Materialized Views` --> maybe too on the nose?

or

`Using Materialized Views in your dbt workflow`

File: website/blog/2023-08-01-announcing-materialized-views.md
Comment (amychen1776) [reply website/blog/2023-08-01-announcing-materialized-views.md:21]: I'll check with the account team. I remember a linkedin post about it but should validate

File: website/blog/2023-08-01-announcing-materialized-views.md
Comment (amychen1776) [reply website/blog/2023-08-01-announcing-materialized-views.md]:
```suggestion
We are only supporting dynamic tables on Snowflake, not Snowflake’s materialized views (for a comparison between Snowflake Dynamic Tables and Materialized Views, refer [docs](https://docs.snowflake.com/en/user-guide/dynamic-tables-comparison#dynamic-tables-compared-to-materialized-views). Dynamic tables are better suited for continuous transformations due to functionality like the ability to join, union, and aggregate on base tables, views , and other dynamic tables. Due to those features, they are also more aligned with what other data platforms are calling Materialized Views. For the sake of simplicity, when I refer to materialized views in this blog, I mean dynamic tables in Snowflake.
```

Comment (amychen1776) [reply website/blog/2023-08-01-announcing-materialized-views.md]: eep didn't copy over

File: website/blog/2023-08-01-announcing-materialized-views.md
Comment (amychen1776) [reply website/blog/2023-08-01-announcing-materialized-views.md:213]: Where can we send them? We don't have a dedicated slack channel for streaming use cases, etc?

File: website/blog/2023-08-01-announcing-materialized-views.md
Comment (amychen1776) [reply website/blog/2023-08-01-announcing-materialized-views.md:21]: got approval from the account team

File: website/blog/2023-08-01-announcing-materialized-views.md
Comment (jasnonaz) [reply website/blog/2023-08-01-announcing-materialized-views.md]: I like `Using Materialized Views in your dbt workflow` , wonder if we can give it some color like `Speed and performance - Using Materialized Views in your dbt workflow`, not exactly that but if we could add in a _little_ hook would be great, if not `Using Materialized Views in your dbt workflow` should work.

Comment (amychen1776) [reply website/blog/2023-08-01-announcing-materialized-views.md]: What about - `A new tool to optimize your dbt workflow: Materialized Views?`

or `Optimizing Materialized Views for the dbt workflow`?

Speed and performance don't feel like the right call here but making MVs is the right call. Like I view dbt as the guardrails you need for MVs

Comment (amychen1776) [reply website/blog/2023-08-01-announcing-materialized-views.md]:
```suggestion
title: "Optimizing Materialized Views with dbt"
```

File: website/blog/2023-08-01-announcing-materialized-views.md
Comment (amychen1776) [inline website/blog/2023-08-01-announcing-materialized-views.md]:
```suggestion
date: 2023-08-03
```

**Review by jasnonaz — APPROVED** [review]
😎

File: website/blog/2023-08-01-announcing-materialized-views.md
Comment (amychen1776) [inline website/blog/2023-08-01-announcing-materialized-views.md]:
```suggestion
{{
config(
       materialized='materialized_view',
)
}}
```

File: website/blog/2023-08-01-announcing-materialized-views.md
Comment (amychen1776) [inline website/blog/2023-08-01-announcing-materialized-views.md]:
```suggestion
     materialized='materialized_view',
```

File: website/blog/2023-08-01-announcing-materialized-views.md
Comment (amychen1776) [inline website/blog/2023-08-01-announcing-materialized-views.md]:
```suggestion
{{
config(
    materialized = 'dynamic_table',
    snowflake_warehouse = '<warehouse>',
    target_lag = '<desired_lag>',
    on_configuration_change = 'apply',
)
}}
```

File: website/blog/2023-08-01-announcing-materialized-views.md
Comment (amychen1776) [inline website/blog/2023-08-01-announcing-materialized-views.md]:
```suggestion
{{
config(
    materialized = 'dynamic_table',
    snowflake_warehouse = '<warehouse>',
    target_lag = '<desired_lag>',
    on_configuration_change = 'apply',
)
}}
```

**Pull request merged**
